.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Special notes for your reviewer**:
+
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
+
+**Release note**:
```

Comment:
+1 @Fryguy I think we can remove the release note bit.

Reply:
Do you mean remove the `release note`? Keeping release notes can make the release get all release notes easily via some command or some tools.

Reply:
@gyliu513 I think this is something that we would also want common across all the repositories. Additionally a "release" in this case is actually a manageiq release so I don't think we would publish release notes for this repo in the same way as application release notes.

Reply:
@Fryguy anything to add on this one? Not sure if that was the reasoning you had in mind.

Reply:
Thanks @carbonin I found that most of the projects of kubernetes, istio, knative all have this release notes for all of their repos, so just trying to copy it to this repo.

If you guys think this is useless, I can remove it. @Fryguy @carbonin

Reply:
@gyliu513 Let's remove it from this PR.  We can always get back to this discussion in a future PR.

Reply:
It's fine. I think @Fryguy is not working today, so I'll just merge this and we can discuss more when he sorts out what we're doing for all the manageiq repos.

Reply:
Sorry, @chessbyte I didn't see your comment before merging

Reply:
So do I need to create a PR remove this? @carbonin

Reply:
If you would, that would be great. Thanks @gyliu513

Reply:
Done at https://github.com/ManageIQ/manageiq-pods/pull/439/files